### PR TITLE
Rearrange documentation

### PR DESCRIPTION
**Pull request by vlrpl**
Introduces some modifications to the documentation (mostly cosmetic) aiming at improving the navigation.
Depends on #460.

### CODE REVIEW

**Review by atenart — CHANGES_REQUESTED** [review]
How do you think we should proceed here? This can't be merged after #460 (would be after the release + the release commit was reordered).

Two possibilities IMO:
- Removing the release commit from this one (and maybe all non-doc commits). Then it can be merged before #460.
- I can cherry-pick your commits in #460.

(Did not propose to close #460 and use this one instead as I'd like to keep a chance for me to put last minute fixes before releasing).

File: mkdocs.yml [hunk 1]
```diff
@@ -8,23 +8,26 @@ repo_url: https://github.com/retis-org/retis
...
+        - Overview: index.md
+        - Collectors:
+            - skb: modules/skb.md
+            - skb-drop: modules/skb_drop.md
+            - skb-tracking: modules/skb_tracking.md
+            - ovs: modules/ovs.md
+            - ct: modules/ct.md
+            - nft: modules/nft.md
+        - Profiles: profiles.md
+        - Filtering: filtering.md
```
[the remark this answers is not in the document]
Comment (atenart) [reply mkdocs.yml:21]: Reason is 1) it's more useful IMO and 2) profiles should always come last from a cli use perspective as they reuse all the cli args.

File: mkdocs.yml [hunk 2]
```diff
@@ -8,23 +8,26 @@ repo_url: https://github.com/retis-org/retis
...
+        - Event sections: event_sections.md
```
[the remark this answers is not in the document]
Comment (vlrpl) [reply mkdocs.yml:22]: let's see if we can make it the collector starting page

File: mkdocs.yml [hunk 3]
```diff
@@ -8,23 +8,26 @@ repo_url: https://github.com/retis-org/retis
...
+        - Python bindings: python.md
+        - Limitations: limitations.md
+        - External resources: resources.md
+    - Run Retis:
```
[the remark this answers is not in the document]
Comment (vlrpl) [reply mkdocs.yml:26]: I like the way this is cut. 
I didn't want to change much, but just group things a little more logically, but this is probably better.

File: mkdocs.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (atenart) [reply mkdocs.yml:21]: Isn't that the same as $topic, moving filtering before profiles? :smile:

**Comment by vlrpl** [comment]
> How do you think we should proceed here? This can't be merged after #460 (would be after the release + the release commit was reordered).
> 
> Two possibilities IMO:
> 
>     * Removing the release commit from this one (and maybe all non-doc commits). Then it can be merged before [[WIP] Release v1.5 #460](https://github.com/retis-org/retis/pull/460).
> 
>     * I can cherry-pick your commits in [[WIP] Release v1.5 #460](https://github.com/retis-org/retis/pull/460).
> 
> 
> (Did not propose to close #460 and use this one instead as I'd like to keep a chance for me to put last minute fixes before releasing).

I think the best way is for you to cherry-pick mine. I avoided pushing on your PR (I think it should be feasible) to avoid any potential mess (also because it's not polite :)
Let me incorporate the additional changes from our discussion and feel free to pick them.

**Comment by atenart** [comment]
> I think the best way is for you to cherry-pick mine. I avoided pushing on your PR (I think it should be feasible) to avoid any potential mess (also because it's not polite :) Let me incorporate the additional changes from our discussion and feel free to pick them.

Sounds good, thanks!

File: mkdocs.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (vlrpl) [reply mkdocs.yml:21]: yes :) but in `index.md` both filtering and tracking come after profiles, so moving them up (or moving profiles after them :) would be "more in line" with `mkdocs.md`.

[the remark this answers is not in the document]
Comment (atenart) [reply mkdocs.yml:21]: Ahhh sorry must have read too quickly... Yes, I think that is a good idea!

File: mkdocs.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (vlrpl) [reply mkdocs.yml:22]: unfortunately, it seems we cannot do:

```
 - Collectors: event_sections.md
     - ...
 ```

I guess the second best option is putting it inside Collectors:

File: mkdocs.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (vlrpl) [reply mkdocs.yml]: I `s/External resources/External/ ` to avoid the repetition. Other than that and minor details I kept the structure you proposed

**Comment by atenart** [comment]
Thanks for the updates, I cherry-picked all your commits in #460.

**Pull request closed without merging**